cmd/validator/validator.go

```diff
@@ -80,16 +82,36 @@ func getFlags() (validatorConfig, error) {
 		return validatorConfig{}, errors.New("Wrong parameter value for reporter, only supports standard or json")
 	}
 
+	if depthPtr != nil && isFlagSet("depth") && *depthPtr < 0 {
```

Comment:
Do we want to create a constant out of "depth" ? Since we use it in two places. I do not see it changing and wouldn't mind retaining it as it is now.